v20.2/backup.md

```diff
@@ -111,13 +111,19 @@ For an example of an incremental backup, see the [Create incremental backups](#c
 
 The `BACKUP` process minimizes its impact to the cluster's performance by distributing work to all nodes. Each node backs up only a specific subset of the data it stores (those for which it serves writes; more details about this architectural concept forthcoming), with no two nodes backing up the same data.
 
-For best performance, we also recommend always starting backups with a specific [timestamp](timestamp.html) at least 10 seconds in the past. For example:
+`BACKUP`, like any read, cannot export a range if it contains an unresolved intent. If CockroachDB hits an intent during iteration, CockroachDB will typically return a `WriteIntentError` and let the store resolve it and retry the request before replying to the sender.
```

Comment:
is "intent" too jargon-y/internal detail for the backup docs? if so, could say "pending / uncommitted write from another transaction" or something?